docker-compose.yml

```diff
@@ -1,7 +1,31 @@
-MongoDB:
-  image: sameersbn/mongodb:latest
-  ports:
-    - "27017:27017"
-  volumes:
-    - /srv/docker/mongodb:/var/lib/mongodb
-  restart: always
+version: '3'
+
+services:
+  adopteitor_api:
+    env_file:
+      - .env
+    container_name: adopteitor_api
+    image: adopteitor-api
+    build: .
+    ports:
+      - "3030:3030"
+    volumes:
+      - .:/var/adopteitor-api
```

Comment:
This "mounts" the current directory (app's root) to `/var/adopteitor-api` in the container.

Meaning that any file read from the app will hit our local HDD files.